### DISCUSSION TITLE
这个有哪些功能？

### DISCUSSION THREAD

**Discussion by SpringCao** [mamoe/mirai#823] [Q&A] [answered]
主页是否可以介绍一下使用方法

**Comment by notnotype** [comment]
# 不能
主页是劝退用户用的

**Comment by Him188** [comment, marked as answer]
这是库，主页有开发文档，有支持的协议列表介绍

**Comment by SpringCao** [comment]
没有使用介绍吗？比如说在终端输入一个命令就可以直接使用


发自我的iPhone

------------------ 原始邮件 ------------------
发件人: Him188 <notifications@github.com&gt;
发送时间: 2021年1月7日 21:09
收件人: mamoe/mirai <mirai@noreply.github.com&gt;
抄送: SpringCao <1580730107@qq.com&gt;, Author <author@noreply.github.com&gt;
主题: 回复：[mamoe/mirai] 这个有哪些功能？ (#823)





 
这是库，主页有开发文档，有支持的协议列表介绍

**Reply by Him188** [reply]
这不是应用，这是库。没有实际功能。

**Comment by notnotype** [comment]
你可能需要这个
```shell
curl -LO "https://github.com/iTXTech/mirai-console-loader/releases/download/v1.0.3/mcl-1.0.3.zip"
unzip mcl*.zip -d mcl
cd mcl
./mcl
```

**Comment by SpringCao** [comment]
感谢


发自我的iPhone

------------------ 原始邮件 ------------------
发件人: Him188 <notifications@github.com&gt;
发送时间: 2021年1月7日 21:15
收件人: mamoe/mirai <mirai@noreply.github.com&gt;
抄送: SpringCao <1580730107@qq.com&gt;, Author <author@noreply.github.com&gt;
主题: 回复：[mamoe/mirai] 这个有哪些功能？ (#823)





 
这不是应用，这是库。没有实际功能。
